documentation/website/src/pages/pysa-playground.js

```diff
@@ -86,16 +87,31 @@ function Results(props) {
   var content = results;
   if (results) {
     if (results.errors !== undefined) {
-      content = results.errors.join('\n');
+      content = results.errors.map((error) => (
+        <div className={classnames(styles.card, styles.error)}>{error}</div>
+      ));
     } else {
-      content = results.data.join('\n');
+      content = results.data.map((result) => {
+        if (result.includes("ERROR"))
```

Comment:
Nit: Although the following should not happen, it might be a good idea to have slightly more strict matching than `ERROR` to ensure we are matching by the prefix of the string, since `ERROR` can match with something like `ERROR123`

```suggestion
        if (result.includes(" ERROR "))
```